src/dropzone.js

```diff
@@ -740,7 +740,7 @@ export default class Dropzone extends Emitter {
   addFile(file) {
     file.upload = {
       // note: this only works if window.isSecureContext is true, which includes localhost in http
-      uuid: self.crypto.randomUUID(),
+      uuid: Window.isSecureContext ? self.crypto.randomUUID() : Dropzone.uuidv4(),
```

Comment:
it must be lowercase `window`, mustn't it?

Reply:
Yes. I'm lost in plain JS.